.github/ISSUE_TEMPLATE/ask-question.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Question
+about: Ask a question about knative/client
+title: ''
+labels: kind/question
```

Comment:
Nice nice @navidshaikh. Love this.

One quick question is whether the labels should also match the labels we are planning for all issues and PRs. How do we see the current defined labels?

Reply:
We can see the current defined labels here https://github.com/knative/client/labels

Reply:
I will add the labels referenced in this PR and missing from repo.

Reply:
Added/updated the labels https://github.com/knative/client/labels PTAL

Reply:
Thank you.